### PR TITLE
feature: Added Hebrew translations

### PR DESCRIPTION
**Pull request by orenk86**

### CODE REVIEW

**Comment by orenk86** [comment]
Okay, rc-calendar and rc-pagination libraries have been updated and everything seems okay. This PR is ready as far as I'm concerned 😊

**Comment by orenk86** [comment]
Worth mentioning that the failing tests seem unrelated to this PR. They also fail locally for me, with the same errors.
When I run `npm test -- -u` the snapshots get updated and all of the tests pass locally for me.

Should I include an update for the snapshots in this PR? Seems a bit unrelated.
Perhaps it would be better to open another one...

File: docs/react/i18n.en-US.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Supported languages:
 
 |Language|Filename|
 |---|---|
+|Hebrew|he_IL|
```
Comment (zombieJ) [inline docs/react/i18n.en-US.md]: Pls order with `Filename` field.

**Review by zombieJ — CHANGES_REQUESTED** [review]

**Comment by orenk86** [comment]
@afc163 @zombieJ May I ask when this will be included in a release? Thanks!

**Pull request merged**